Command/Message/schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "https://smart-data-models.github.io/datamodel.AutonomousMobileRobot/Command/Message/schema.json",
+  "type": "object",
+  "properties": {
+    "type": { "$ref": "https://fiware.github.io/data-models/specs/Robot/schema.json#/definitions/type" },
+    "time": {
+      "$ref": "https://fiware.github.io/data-models/specs/Robot/schema.json#/definitions/time",
+      "description": "Send time"
+    },
+    "command": { "$ref": "https://fiware.github.io/data-models/specs/Robot/AutonomousMobileRobot/schema.json#/definitions/command" },
+    "waypoints": { "$ref": "https://fiware.github.io/data-models/specs/Robot/AutonomousMobileRobot/schema.json#/definitions/waypoints" }
+  },
+  "required": ["id", "type", "time", "command", "waypoints"],
+  "additionalProperties": true
```

Comment:
it should be added the required clause including just id and type. then it is not required to use addtionalProperties clause